build.gradle

```diff
@@ -4,16 +4,19 @@ buildscript {
     }
 }
 
-group 'pap.lorinc'
+group 'birthdayQuiz'
 version '1.0.0'
 
 apply plugin: 'groovy'
 apply plugin: 'java'
-
+apply plugin: 'application'
 apply plugin: 'idea'
 
 sourceCompatibility = JavaVersion.VERSION_1_8
 
+mainClassName = 'birthdayQuiz.BirthdayQuiz'
+applicationDefaultJvmArgs = ['-Dfile.encoding=UTF-8'] // TODO not yet working 
```

Comment:
needs to be started as `chcp 65001 && BirthdayQuiz.bat` in order to show all Hungarian characters .. it's the only workaround I could find :/